plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -339,7 +340,6 @@ function FakeCover:init()
     }
 end
 
-
```

Comment:
Some double blank lines are not meaningless. I guess I added these to separate a bit more different classes: ItemShortcut / FakeCover / MosaicMenuItem.
(So, don't touch what you don't need to, unless it's really crappy/unreadable :)

Reply:
Okay, restored that blank line.